static/bidder-params/bliink.json

```diff
@@ -4,5 +4,14 @@
   "description": "A schema which validates params accepted by the Bliink adapter",
   "type": "object",
 
-  "properties": {}
+  "properties": {
```

Comment:
This JSON schema documents that two possible properties exist, and if they do their values must be strings. But they do not need to be present in a request. If this is correct, then all is good. If there are further requirements, we should probably discuss how best to get them in.

Reply:
The field TagId is required, but not the imageUrl field.

Reply:
I would suggest some code in your adapter to check for the existence of TagId in the request, and return an error if it is not found. Then, at a later date put in a PR that sets TagId to required in the JSON schema. PBS made a design choice that if any of the adapter parameters fails the schema check, the whole request fails.

The reasoning for this strictness was the adapter parameter schema will rarely change, if ever. However a publisher with a typo in one of the adapter's parameter may not notice anything amiss if at least some prebid adapters are returning ads. So failing completely is more likely to get the publisher's attention to fix the typo.

Once enough time has passed for publishers to fix their prebid configs for your new rrquirements, you can set your schema to flag TagId as required and have the full benefit of the schema enforcement.

Reply:
Done.
I directly set the field "tagId" to required because for the moment, we don't have any publishers using our adapter.